include/miopen/miopen.h

```diff
@@ -5324,6 +5324,7 @@ typedef enum
     miopenTensorSoftmaxDY    = 14,
 #endif
 
+    miopenScalarArgument = 1 << 16,
```

Comment:
[Q] How does the user specify the datatype of the scalar argument? Implied by the enum member?

Reply:
Since this PR doesn't introduce any scalar parameters on its own, user technically doesn't have any way to pass one. But the parameters are meant to be added as `someScalarArg = miopenTensorArgumentIsScalar | someNumber`, so user just calls it by `someScalarArg`. Separate flag is kind of an implementation detail that has no reason to hide it as it makes it obvious, which parameters are scalars. Datatype is inferred from name as of now, but in theory we could support e.g. creating and passing tensor descriptor with size 1 and some metadata (like type) for scalars.